packages/flutter/test/material/buttons_test.dart

```diff
@@ -82,6 +84,34 @@ void main() {
     );
 
     expect(tester.getSize(find.byType(FlatButton)), equals(const Size(88.0, 36.0)));
+    // Scaled text rendering is different on Linux and Mac by one pixel.
```

Comment:
please make sure to continue to mention the URL to the relevant bug whenever working around this bug.

Reply:
Done.  (I actually meant to do that originally, but got distracted when looking up the bug... :-))